project/src/components/app/app.tsx

```diff
@@ -1,5 +1,13 @@
-function App(): JSX.Element {
-  return <p>Hello, world!</p>;
+import MainPage from '../../pages/main-page/main-page';
+
+type Props = {
+  countOfAvailablePlaces: number;
+};
+
+function App({countOfAvailablePlaces}:Props): JSX.Element {
+  return (
+    <MainPage countOfAvailablePlaces = {countOfAvailablePlaces}/>
```

Comment:
Форматирование должно быть такое
```suggestion
    <MainPage countOfAvailablePlaces={countOfAvailablePlaces} />
```